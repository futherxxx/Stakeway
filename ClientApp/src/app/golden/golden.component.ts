import { Component, OnInit, ViewChild } from '@angular/core';

@Component({
  selector: 'app-golden',
  templateUrl: './golden.component.html',
  styleUrls: ['./golden.component.scss']
})
export class GoldenComponent implements OnInit {
  @ViewChild('gameChild') child;

  constructor() { }
  fileText:any='Golden Chance Lotto';
  ngOnInit() {
  }
  ngAfterViewInit() {
    this.child.getAllGames();
 }
}
