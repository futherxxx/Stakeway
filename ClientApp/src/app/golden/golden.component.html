<app-games [fileText]="fileText" #gameChild></app-games>